accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloInstance.scala

```diff
@@ -50,6 +51,18 @@ object AccumuloInstance {
     val tokenClass = token.getClass.getCanonicalName
     BaseAccumuloInstance(instanceName, zookeeper, user, (tokenClass, tokenBytes))
   }
+
+  def apply(uri: URI): AccumuloInstance = {
+    import geotrellis.util.UriUtils._
+
+    val zookeeper = uri.getHost
+    val instance = uri.getPath.drop(1)
+    val (user, pass) = getUserInfo(uri)
+    AccumuloInstance(
+      instance, zookeeper,
+      user.getOrElse("root"),
+      new PasswordToken(pass.getOrElse("")))
```

Comment:
Can Accumulo has an empty string as a pwd? May be an exception can take place here?